prody/dynamics/plotting.py

```diff
@@ -211,7 +211,7 @@ def showProjection(ensemble, modes, *args, **kwargs):
       * 3 modes: :meth:`~mpl_toolkits.mplot3d.Axes3D.plot`"""
 
     import matplotlib.pyplot as plt
-
+    plt.figure()
```

Comment:
I will do it then. In case you need to find it later, you can start from a function called confProDy